### PR TITLE
Add an example on how to use a custom st2web config

### PR DESCRIPTION
**Pull request by Kami**
This pull request adds an example on how to use a custom st2web config file.

### CODE REVIEW

File: files/config.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Extreme Networks, Inc.
```
Comment (armab) [inline files/config.js:1]: Need to change it to the StackStom Authors. Check the `stackstorm/st2` codebase for example.

File: docker-compose.yml [hunk 2]
```diff
@@ -20,6 +20,9 @@ services:
       retries: 1
     volumes:
       - ./scripts/st2web-healthcheck.sh:/st2web-healthcheck.sh
+      # If you want to use a custom st2web config, edit ./files/config.js accordingly and
```
Comment (armab) [inline docker-compose.yml:23]: Can we include a small section in the README as well? That'll be helpful for the broader community.

**Review by armab — CHANGES_REQUESTED** [review]
Thanks!

I left a couple of comments. Additionally, we'll need a CHANGELOG record for this PR to indicate the enhancement.

File: files/config.js (hunk 1 above)
Comment (Kami) [reply files/config.js:1]: I just copied existing file content, but I can update it.

Comment (armab) [reply files/config.js:1]: Yes, that would be great. It means we forgot to update those files. We however made changes to `st2` and `st2flow` codebases.

**Comment by armab** [comment]
Yeah, having a mention in the Changelog is helpful as it really brings pointers and docker-compose changes about configuring st2web endpoints.

**Comment by Kami** [comment]
Updated readme, changelog, license headers.

File: files/config.js (hunk 1 above)
Comment (Kami) [reply files/config.js:2]: Is that what you are referring to - https://github.com/StackStorm/st2flow/commit/2bde34a0bd0aa022150b81f710d8f4409d631180?

It seems like it's still missing "Copyright 2020 The StackStorm Authors." part. Or we don't want that anymore?

Comment (armab) [reply files/config.js:2]: It's like https://github.com/StackStorm/st2web/pull/759/files#diff-cb21033c4c4589a654b297f30aee73f6fac94040264a0acb09e816de1251d8e7R1

**Review by armab — APPROVED** [review]
:+1: Thanks!

**Pull request merged**